src/main.rs

```diff
@@ -130,9 +130,19 @@ fn user_confirm(message: String) -> bool {
 /// If either fails, it prints an error and exits.
 /// On success, it returns a `ResolvedIds` structure that contains ids to avoid needing to resolve
 /// the names again.
-fn resolve_ids(org_id: Option<&str>, env: Option<&str>, proj: Option<&str>) -> Result<ResolvedIds> {
+fn resolve_ids(org_id: Option<&str>, config: &Config) -> Result<ResolvedIds> {
     // The `err` value is used to allow accumulation of multiple errors to the user.
     let mut err = false;
+    let env = if !config.environment.is_empty() {
```

Comment:
I'd just make the `Config` struct type the `environment` value as `Option<String>` and deal with empty strings when constructing it. The config instance is accessible globally, so it can be a bit confusing that there are two different "rules" applied to the same data, depending on the caller.